src/widgets/dnshole/DnsHoleSummary.tsx

```diff
@@ -70,14 +83,32 @@ function AdHoleSummaryWidgetTile({ widget }: AdHoleSummaryWidgetProps) {
       </Card>
       <Card
         m="xs"
-        bg={widget.properties.usePiHoleColors ? 'rgb(177, 114, 12)' : 'rgb(255 165 20)'}
+        sx={(theme) => {
+          if (!widget.properties.usePiHoleColors) {
+            return {};
+          }
+
+          if (theme.colorScheme === 'dark') {
+            return {
+              backgroundColor: 'rgba(255, 165, 20, 0.4)',
+            };
+          }
+
+          return {
+            backgroundColor: 'rgba(255, 165, 20, 0.4)',
+          };
+        }}
         withBorder
       >
         <Center h="100%">
           <Stack align="center" spacing="xs">
             <IconPercentage size={30} />
             <div>
-              <Text align="center">{(data.adsBlockedTodayPercentage * 100).toFixed(2)}%</Text>
+              <Text align="center">
+                {t('card.metrics.adsBlockedTodayPercentage', {
+                  percentage: (data.adsBlockedTodayPercentage * 100).toFixed(2),
+                })}
+              </Text>
```

Comment:
I actually meant something else. This change can be reverted.

```suggestion
              <Text align="center">{(data.adsBlockedTodayPercentage * 100).toFixed(2)}%</Text>
```